File: API/API/ViewModel/TopicStatistic.cs
using API.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace API.ViewModel
{
    public class TopicStatistic
    {
        public Topic Topic { get; set; }
        public List<Statistic> Statistics { get; set; }
    }
}

File: Android_Final/app/src/main/java/com/example/android_etpj/models/TopicStatistic.java
package com.example.android_etpj.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class TopicStatistic implements Serializable {
    @SerializedName("Topic")
    private  Topic topic;

    @SerializedName("Statistics")
    private List<Statistic> statistics;

    public TopicStatistic() {
    }

    public Topic getTopic() {
        return topic;
    }

    public void setTopic(Topic topic) {
        this.topic = topic;
    }

    public List<Statistic> getStatistics() {
        return statistics;
    }

    public void setStatistics(List<Statistic> statistics) {
        this.statistics = statistics;
    }
}
